tests/integration/components/test_render_cookies_banner.py

```diff
@@ -0,0 +1,10 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestRenderCookiesBanner(IntegrationTestCase):
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey("test_dropdown_mandatory")
+
+    def test_cookies_banner_renders(self):
+        self.assertInBody("Tell us whether you accept cookies")
```

Comment:
I think we should rename this to `TestCookieBanner` and the test to `test_cookies_banner_contents` as there is no rendering as integration tests don't use JS.